pointcloud/basic_octree_test.go

```diff
@@ -130,16 +137,22 @@ func TestBasicOctreeSet(t *testing.T) {
 		basicOct, err := createNewOctree(center, side)
 		test.That(t, err, test.ShouldBeNil)
 
-		err = basicOct.Set(r3.Vector{X: 0, Y: 0, Z: 0}, NewValueData(1))
+		err = basicOct.Set(r3.Vector{X: 0, Y: 0, Z: 0}, NewValueData(3))
 		test.That(t, err, test.ShouldBeNil)
+		mp := basicOct.MaxProb()
+		test.That(t, mp, test.ShouldEqual, .03)
 
-		err = basicOct.Set(r3.Vector{X: -.5, Y: 0, Z: 0}, NewValueData(1))
+		err = basicOct.Set(r3.Vector{X: -.5, Y: 0, Z: 0}, NewValueData(2))
 		test.That(t, err, test.ShouldBeNil)
+		mp = basicOct.node.children[0].MaxProb()
+		test.That(t, mp, test.ShouldEqual, .03)
 
-		err = basicOct.Set(r3.Vector{X: -.4, Y: 0, Z: 0}, NewValueData(1))
+		err = basicOct.Set(r3.Vector{X: -.4, Y: 0, Z: 0}, NewValueData(4))
```

Comment:
[nit] Consider defining d1,d2,d3 so that we can do:
```
d1 := 2
err = basicOct.Set(r3.Vector{X: 0, Y: 0, Z: 0}, NewValueData(d1))
...
test.That(t, mp, test.ShouldEqual, d1/100.)
```

or even 

`test.That(t, mp, test.ShouldEqual, math.Max(d1,d2,d3)/100)`

Reply:
can be applied to most tests